typescript/sdk/src/agents/types.ts

```diff
@@ -76,12 +76,6 @@ export function buildAgentConfig(
       },
       protocol: 'ethereum',
       finalityBlocks: metadata.blocks?.reorgPeriod ?? 1,
-      connection: {
```

Comment:
Iirc I added this mostly as a workaround in Helm to support httpQuorum but I'm not totally sure. Just flagging in case there are weird issues when deploying that this may be needed